tests/example_workflows/pyxaid2/example_1_Si/step1/submit.slm

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 #SBATCH --partition=valhalla  --qos=valhalla
 #SBATCH --clusters=chemistry
-#SBATCH --time=100:00:00
-#SBATCH --nodes=2
+#SBATCH --time=300:00:00
```

Comment:
this part is not necessary in the commit, you can keep it uncommitted local version. I can accept this change, but it is more of a time-to-time user-specific setting and need not be committed.